test/recipes/30-test_pairwise_fail.t

```diff
@@ -63,3 +63,17 @@ SKIP: {
                  "-pairwise", "dsakat", "-dsaparam", data_file("dsaparam.pem")])),
        "fips provider dsa keygen kat failure test");
 }
+
+SKIP: {
+    skip "Skip EDDSA test because of no ecx in this build", 2
+        if disabled("ec") || disabled("ecx");
```

Comment:
`if disabled("ecx");` should be sufficient here.

Reply:
Revert changes and fix test via correcting the second argument of skip statement.